tests/pytest.ini

```diff
@@ -1,4 +1,6 @@
 [pytest]
+timeout = 10
+timeout_method = thread
```

Comment:
Ensuring the tests won't freeze in case of a deadlock :sweat_smile:

Note that default timeout-method is `signal`, but it doesn't appear to handle deadlocks well.